tests/1.8/regression/metadata/fail/stream-undefined-id/metadata

```diff
@@ -0,0 +1,49 @@
+/* CTF 1.8 */
+trace {
+	major = 1;
+	minor = 8;
+	uuid = "ff29057a-fe6b-4996-9d98-06f95dcd55cf";
```

Comment:
check metadata-minimal-accepted/metadata from ctf-testsuite. uuid is not needed.